tests/Makefile.am

```diff
@@ -113,4 +113,12 @@ EXTRA_DIST =                                                           \
  test_read_hmac-sha1-96                                                \
  test_read_hmac-sha2-256                                               \
  test_read_hmac-sha2-512                                               \
- test_read_rijndael-cbc
+ test_read_hmac-sha2-512.bat                                           \
+ test_read_rijndael-cbc                                                \
+ test_read_rijndael-cbc.bat                                            \
+ test_read_hmac-sha1-etm                                               \
+ test_read_hmac-sha1-etm.bat                                           \
+ test_read_hmac-sha2-256-etm                                           \
+ test_read_hmac-sha2-256-etm.bat                                       \
+ test_read_hmac-sha2-512-etm                                           \
+ test_read_hmac-sha2-512-etm.bat
```

Comment:
We can delete the `.bat` files. After [making portable](05c7d128c8966f60ec168790432e6e76e91db9db) how CMake executes these tests, we no longer use these and existing ones are already deleted.